### PR TITLE
TLM 1 and High Level Trace issues

### PR DESCRIPTION
**Pull request by maxiwell**

### CODE REVIEW

File: env.sh.in [hunk 1]
```diff
@@ -12,3 +13,21 @@ export PATH="$exec_prefix/bin:$PATH"
...
+    else
+        echo "Couldn't locate the top of the ArchC source code."
+    fi
+}
+
+function acprefix {
+    if [ "$prefix" ]; then
+        cd "$prefix"
+    else
+        echo "Couldn't locate the installation path of the ArchC."
```
Comment (rafaelauler) [inline env.sh.in:29]: nit: Couldn't locate the ArchC installation path.

File: src/aclib/ac_tlm/ac_tlm_port.H [hunk 2]
```diff
@@ -92,8 +92,8 @@ public:
    * @param wordsize Word size in bits.
    * 
    */
-  virtual void read(ac_ptr buf, uint32_t address,
-		    int wordsize);
+  virtual void read(ac_ptr buf, uint32_t address,  int wordsize);
+  virtual void read(ac_ptr buf, uint32_t address, int wordsize,sc_core::sc_time &time_info,unsigned int procId = 0);
```
Comment (rafaelauler) [inline src/aclib/ac_tlm/ac_tlm_port.H:96]: nit: I observed that the spaces are uneven. I would suggesting using a tool such as clang-format to automatically format the code. It has vim integration and it is easy to use.

File: src/aclib/ac_tlm/ac_tlm_port.H [hunk 3]
```diff
@@ -104,8 +104,8 @@ public:
    * @param n_words Number of words to be read.
    * 
    */
-  virtual void read(ac_ptr buf, uint32_t address,
-		    int wordsize, int n_words);
+  virtual void read(ac_ptr buf, uint32_t address,  int wordsize, int n_words);
+  virtual void read(ac_ptr buf, uint32_t address, int wordsize, int n_words,sc_core::sc_time &time_info,unsigned int procId = 0);
```
Comment (rafaelauler) [inline src/aclib/ac_tlm/ac_tlm_port.H:108]: Ditto

File: src/aclib/ac_tlm/ac_tlm_port.cpp [hunk 4]
```diff
@@ -103,6 +103,10 @@ void ac_tlm_port::read(ac_ptr buf, uint32_t address, int wordsize) {
   }
 }
 
+void ac_tlm_port::read(ac_ptr buf, uint32_t address, int wordsize,sc_core::sc_time &time_info,unsigned int procId ){
```
Comment (rafaelauler) [inline src/aclib/ac_tlm/ac_tlm_port.cpp:106]: nit: Spacing..

**Comment by maxiwell** [comment]
Split this PR in 3 PR.

**Pull request closed without merging**